Sources/NIOConcurrencyHelpers/atomics.swift

```diff
@@ -389,9 +401,11 @@ extension UInt: AtomicPrimitive {
     public static let atomic_store                = catmc_atomic_unsigned_long_store
 }
 
-/// `AtomicBox` is a heap-allocated box which allows atomic access to an instance of a Swift class.
+/// `AtomicBox` is a heap-allocated box which allows lock-free access to an instance of a Swift class.
 ///
-/// It behaves very much like `Atomic<T>` but for objects, maintaining the correct retain counts.
+/// - warning: The use of `NIOSpinLockBox` should be avoided because it requires an implementation of a spin-lock
```

Comment:
`NIOSpinLockBox` isn't a type that actually exists.

Reply:
ta, fixed